migration.go

```diff
@@ -14,7 +14,7 @@ import (
 
 type MigrationRecord struct {
 	VersionId int64
-	TStamp    time.Time
+	TStamp    string
```

Comment:
Thanks for your contribution anyway. I'd be glad if you made this working with custom Unmarshaler on your set-up. I'm not quite clear why it doesn't work right away.

```diff
- TStamp time.Time
+ TStamp Time
```

```go
type Time time.Time

func (t Time) MarshalDB() (interface{}, error) {
	return //...
}

func (t Time) UnmarshalDB(in interface{}) error {
	// ...
	return nil
}
```